### DISCUSSION TITLE
Error getting Snapchat Package Info

### DISCUSSION THREAD

**Discussion by flawnn** [rodit/SnapMod#199] [Q&A/Support] [answered]
Hey everyone,
SnapMod not installing and working whyever. I am on Android 13 Beta 3, Lsposed Zygisk 1.8.3 as well as activated for SnapMod (but only Snapchat).

No module logs, seems not to even run.

EDIT: It seems to try to load the app hooks after FCing, sending some logs.

**Comment by flawnn** [comment]
[LSPosed_2022-06-14T19_55_45.111394.zip](https://github.com/rodit/SnapMod/files/8902689/LSPosed_2022-06-14T19_55_45.111394.zip)
Here the log is!

**Comment by flawnn** [comment, marked as answer]
Got it, module needed some minutes to load

**Reply by fgjaee** [reply]
I'm having this issue after play store updated snapchat automatically. I am back to the correct version. How did you fix?

**Reply by fgjaee** [reply]
> I'm having this issue after play store updated snapchat automatically. I am back to the correct version. How did you fix? 

Edit: long press on the module and re-optimize did it for me. Thank you.
